hudi-common/src/main/java/org/apache/hudi/common/model/HoodieWriteStat.java

```diff
@@ -135,6 +135,7 @@
   /**
    * Total number of rollback blocks seen in a compaction operation.
    */
+  @Nullable
```

Comment:
Please avoid this change as part of this diff

Reply:
Discussed offline. Without this we are not able to read certain archived commits